### PR TITLE
fix #65716: Import InstrumentChange from MusicXML

### PR DESCRIPTION
**Pull request by Igevorse**
1. Fix for Mixer entries and multiple InstrumentChanges: don't add InstrumentChange with Instrument same as previous.
2. Fix for sound: added Channel::updateInitList() call.

### CODE REVIEW

File: mscore/importxml.cpp [hunk 1]
```diff
@@ -108,6 +108,7 @@
 #include "importxmlfirstpass.h"
 #include "libmscore/instrchange.h"
 #include "importmxml.h"
+#include "mscore/seq.h"
```
Comment (lasconic) [inline mscore/importxml.cpp]: is that really needed? Why ?

Comment (Igevorse) [inline mscore/importxml.cpp]: @lasconic No, I forgot to remove this line after clearing the code.

**Comment by lasconic** [comment]
Do you have a test case? Could you add it as a mtest?

**Comment by dmitrio95** [comment]
Needs an update, please reopen this PR for this feature if needed.

**Pull request closed without merging**